src/main/java/gamer/Gamer.java

```diff
@@ -0,0 +1,19 @@
+/**
+* @Package_name   : gamer
+* @Interface_name     : BaseDAO 
+*
+* Create Date : 2023-10-21 
+* Create User : 정은채
+*/
+
+package gamer;
+
+public interface Gamer {
+	
+	/**
+	* Description : 숫자야구게임 게임참여자의 play 방식
+	*
+	* @Method : play()
+	*/  
+	public void play();
```

Comment:
인터페이스에 선언된 모든 메서드는 기본적으로 `public`이기 때문에, 작성하지 않으셔도 됩니다:)
```suggestion
	void play();
```

Reply:
넵넵!